lib/barbeque/message/job_execution.rb

```diff
@@ -13,7 +13,11 @@ def assign_body(message_body)
         super
         @application = message_body['Application']
         @job  = message_body['Job']
-        @body = message_body['Message']
+        @body = if message_body['Message'].is_a?(String)
+                  JSON.parse(message_body['Message'])
```

Comment:
Basically no. If non-JSON string came, the exception handling (shows the follow) world occur.

https://github.com/cookpad/barbeque/blob/8ea3873a147e1917c9b27d4b9d8dc04922f56672/lib/barbeque/message.rb#L17-L20